scripts/create-latest-svc.sh

```diff
@@ -0,0 +1,130 @@
+#/bin/bash
+
+set -e
+
+#
+# Downloads latest releases (not pre-release) runner
+# Configures as a service
+#
+# Examples:
+# RUNNER_CFG_PAT=<yourPAT> ./create-latest-svc.sh myuser/myrepo
+# RUNNER_CFG_PAT=<yourPAT> ./create-latest-svc.sh myorg
+#
+# Usage:
+#     export RUNNER_CFG_PAT=<yourPAT>
+#     ./create-latest-svc scope [name] [user]
+#
+#      scope required  repo (:owner/:repo) or org (:organization)
+#      name  optional  defaults to hostname
+#      user  optional  user svc will run as. defaults to current
+# 
+# Notes:
+# PATS over envvars are more secure
+# Should be used on VMs and not containers
+# Works on OSX and Linux 
+# Assumes x64 arch
+#
+
+runner_scope=${1}
+runner_name=${2:-$(hostname)}
+svc_user=${3:-$USER}
+
+echo "Configuring runner @ ${runner_scope}"
+sudo echo
```

Comment:
why do we need to `sudo echo`, to avoid prompt downstream?

Reply:
Yeah, I was trying to prompt early so they don't kick it off, look away only to find a prompt later.